mac

```diff
@@ -311,9 +331,7 @@ install_dependencies() {
   fancy_echo "Installing dependencies"
   brew bundle --global --verbose --no-upgrade
 
-  install_rvm
-  install_nvm
-  setup_postgres
+  #setup_postgres
```

Comment:
is there any reason we disable the `setup postgres` @olivierobert

Reply:
Just an omission from my side when testing on Intel as I had already everything installed. Added back in [5583fd1](https://github.com/nimblehq/laptop/pull/34/commits/5583fd10c07a4887b871c5884156cf0997d937a8)